setup.py

```diff
@@ -143,7 +143,7 @@ def get_description():
       test_suite = "tests",
       install_requires = ["numpy>=1.13.1", "awkward>=0.10.0", "uproot-methods>=0.6.0", "cachetools"],
```

Comment:
Shouldn't `xxhash` appear here rather than `tests_require` ?

Reply:
For pip, which is determined by setup.py, lz4 and therefore xxhash are optional dependencies, though we want to test them. For conda, they're strict dependencies, and that's controlled by uproot-feedstock.

Reply:
hmm, well then [this import](https://github.com/scikit-hep/uproot/blob/master/uproot/source/compressed.py#L11) should be wrapped by a `try:` statement if it is optional.  I encountered an error because I didn't have `xxhash` installed.

Reply:
Yes, it absolutely must. @reikdas?

Reply:
Actually, it should be inside of the function call, just like `import lz4` is, and probably right after `import lz4` to indicate the relationship. Although I think `xxhash` is used in a different function... anyway, it should be inside of a function so that it doesn't load on startup if you have it but don't use it.

Reply:
Sorry, this was my mistake. I opened a PR fixing this. Also @jpivarski I have a draft release ready once you have merged it.